src/olympia/reviewers/management/commands/auto_approve.py

```diff
@@ -51,9 +55,13 @@ def add_arguments(self, parser):
         )
 
     def fetch_candidates(self):
```

Comment:
Nite: Would it make sense to rename this to `fetch_candidate_ids`? Maybe that's not necessary.

Reply:
With the adjusted docstring I don't think it's necessary. It's unlikely that method would ever be used outside of this class anyway.